src/components/SendTransaction/ConfirmTransaction.tsx

```diff
@@ -133,90 +82,68 @@ export const ConfirmTransaction = ({
-            </td>
-          </tr>
-          {formData.memoContent ? (
-            <tr>
-              <th>Memo</th>
-              <td>
-                {formData.memoContent} ({getMemoTypeText(formData.memoType)})
-              </td>
-            </tr>
-          ) : null}
-          <tr>
-            <th>Fee</th>
-            <td>{maxFee} lumens</td>
-          </tr>
-        </tbody>
-      </TableEl>
-
-      {!formData.isAccountFunded && (
-        <InfoBlock>
-          The destination account doesn’t exist. A create account operation will
-          be used to create this account.{" "}
-          <TextLink
-            href="https://developers.stellar.org/docs/tutorials/create-account/"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Learn more about account creation
-          </TextLink>
-        </InfoBlock>
-      )}
-
-      {status === ActionStatus.PENDING &&
-        settings.authType &&
-        settings.authType !== AuthType.PRIVATE_KEY && (
-          <InfoBlock>{getInstructionsMessage(settings.authType)}</InfoBlock>
+    <>
+      <Modal.Heading>Confirm transaction</Modal.Heading>
+
+      <Modal.Body>
+        <LabelAndValue label="Sending to address">
+          <Identicon publicAddress={formData.toAccountId} />
+        </LabelAndValue>
+
+        {formData.isAccountUnsafe && <AccountIsUnsafe />}
+
+        <LabelAndValue label="Amount">
+          {formData.amount}{" "}
+          {new BigNumber(formData.amount).eq(1) ? "lumen" : "lumens"}
+        </LabelAndValue>
+
+        {formData.memoContent ? (
+          <LabelAndValue label="Memo">
+            {formData.memoContent} ({getMemoTypeText(formData.memoType)})
+          </LabelAndValue>
+        ) : null}
+
+        <LabelAndValue label="Fee">{maxFee} lumens</LabelAndValue>
+
+        {!formData.isAccountFunded && (
+          <InfoBlock>
+            The destination account doesn’t exist. A create account operation
+            will be used to create this account.{" "}
+            <TextLink href="https://developers.stellar.org/docs/tutorials/create-account/">
+              Learn more about account creation
+            </TextLink>
+          </InfoBlock>
         )}
-    </ModalContent>
+
+        {status === ActionStatus.PENDING &&
+          settings.authType &&
+          settings.authType !== AuthType.PRIVATE_KEY && (
+            <InfoBlock>{getInstructionsMessage(settings.authType)}</InfoBlock>
+          )}
+      </Modal.Body>
+
+      <Modal.Footer>
+        <Button
+          onClick={handleSend}
+          iconLeft={<Icon.Send />}
+          isLoading={status === ActionStatus.PENDING}
+        >
+          Submit transaction
+        </Button>
+        <Button
+          onClick={onBack}
+          variant={Button.variant.secondary}
+          disabled={status === ActionStatus.PENDING}
+        >
+          Back
+        </Button>
+      </Modal.Footer>
+
+      {status === ActionStatus.PENDING && (
```

Comment:
NIT: since we're checking `status === ActionStatus.PENDING` a few times, might be best to cache as a var